mysite/users/templates/user/profile.html

```diff
@@ -16,6 +16,27 @@
             Profile Image</span>
         </div>
 
+
```

Comment:
@kritikaparmar-programmer why do we need this code snippet from here to [here](https://github.com/harsh-9in/Blogger/pull/175/files#diff-8208737725c422c84328b370f39b3b24R39) when we already have the the same code from [here](https://github.com/harsh-9in/Blogger/pull/175/files#diff-8208737725c422c84328b370f39b3b24R95). Also if you remove the `save` button how will the image get update in the backend and in the user profile?